### PR TITLE
Added a note regarding naming the planning group

### PR DESCRIPTION
**Pull request by aadityasaraiya**
As per this [ROS Answers Question](https://answers.ros.org/question/290983/error-in-planning-group-with-moveit-setup-assistant/), adding a space at the end of planning group name may lead to the "manipulator" group key word to be not detected for adding poses and end-effectors. Hence, updated the documentation to alert newbies like me.

### CODE REVIEW

File: gh_pages/_source/session3/Build-a-Moveit!-Package.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ In this exercise, you will generate a MoveIt package for the UR5 workcell you bu
        type = 'fixed'
        ```
 
-    1. Add a planning group called `manipulator` that names the kinematic chain between `base_link` and `tool0`.
+    1. Add a planning group called `manipulator` that names the kinematic chain between `base_link` and `tool0`. Note: Make sure there are no spaces after the planning group name.
```
Comment (jrgnicho) [inline gh_pages/_source/session3/Build-a-Moveit!-Package.md]: I'd rephrase this as:
```
Make sure there are no space characters before or after the group name
```

Comment (gavanderhoorn) [reply gh_pages/_source/session3/Build-a-Moveit!-Package.md]: Perhaps even more generic: follow ROS naming guidelines/requirements and don't use *any* whitespace, anywhere.

**Comment by jdlangs** [comment]
@aadityasaraiya, if you're feeling ambitious after this contribution, I'm sure it would be a welcome change to add a check inside the MoveIt setup assistant GUI which would reject group names with whitespace, or at least warn about it.

File: gh_pages/_source/session3/Build-a-Moveit!-Package.md (hunk 1 above)
Comment (aadityasaraiya) [reply gh_pages/_source/session3/Build-a-Moveit!-Package.md]: @gavanderhoorn @jrgnicho  Thanks for the comments. I have made the required changes.

**Comment by aadityasaraiya** [comment]
@jdlangs Thanks for the suggestion. That definitely crossed my mind. I was looking into the [planning_groups_widget](https://github.com/ros-planning/moveit/blob/kinetic-devel/moveit_setup_assistant/src/widgets/planning_groups_widget.cpp) file in order to identify where this check can be put up. Will try my best with this and will update if there is any progress.

**Comment by jrgnicho** [comment]
@aadityasaraiya  thanks for the contribution.

**Comment by aadityasaraiya** [comment]
@jdlangs Hey! Just an update. As you had suggested previously, I have added the check for unintentional whitespace and have created [this PR](https://github.com/ros-planning/moveit/pull/919/commits/a41ed7c8e0861fc0e10430ead961d927865e1d9d) for the same in the MoveIt! repo. Thanks!
